x-pack/solutions/security/plugins/security_solution/public/common/components/empty_prompt/empty_prompt.tsx

```diff
@@ -47,7 +47,7 @@ const pageHeaderStyles = css`
 `;
 
 const getFlexItemStyles = (euiTheme: EuiThemeComputed) => css`
-  background: ${euiTheme.colors.lightestShade};
+  background: ${euiTheme.colors.backgroundBaseSubdued};
```

Comment:
It's awesome you're already resolving legacy shade tokens, when it's not required yet 🎉 

One note: These semantic tokens don't map 1:1 to the legacy shade tokens, that's why there is no proper guidance yet.
This means that the colors will change in Amsterdam (mainly dark mode), you want to align with design if that's ok at this point.

For this example `lightestShade` to `backgroundBaseSubdued`

| Token | colorMode | Amsterdam | Borealis |
|-------|------------|-------------|---------|
| `lightestShade` | LIGHT | `#F1F4FA` | `#ECF1F9` |
| `lightestShade` | DARK | `#25262E` | `#172336` |
| `backgroundBaseSubdued` | LIGHT | `#F7F8FC` | `#F6F9FC` |
| `backgroundBaseSubdued` | DARK | `#141519` | `#07101F` |

Reply:
In case, to keep color parity in Amsterdam what you can do is conditionally update colors:

```ts
background: ${isAmsterdam ? euiTheme.colors.lightestShade : euiTheme.colors.backgroundBaseSubdued};
```

Reply:
Amsterdam:
<img width="2557" alt="landing_old_theme" src="https://github.com/user-attachments/assets/6e16bcd0-d539-492e-baea-9f65070b2267" />

Borealis:
<img width="2553" alt="landing_new_theme" src="https://github.com/user-attachments/assets/a7ebde74-51e6-4b9e-b32b-c731d0f00da3" />

Reply:
I'd suggest also looking at the Dark mode, that's where the color values tend to differ more for shades.

Reply:
Amsterdam:
<img width="2560" alt="landing_before" src="https://github.com/user-attachments/assets/098e3c63-051d-4e70-96ef-4ff153e36c91" />

Borealis:
<img width="2556" alt="landing_after" src="https://github.com/user-attachments/assets/b68fcab5-e318-4c7b-9cca-0d2114b4dd1a" />

Reply:
Ah sorry, I wasn't precise enough. 😅 
The difference is in Amsterdam before this token update and with the token update. Amsterdam colors change and the question is if that's ok, if so we don't need to add a condition 🙂 

Generally I'd say Amsterdam dark looks fine, but I'm not the responsible designer here 😅

Reply:
Honestly, as long as nothing is broken (e.g. not legible), we should be ok.
So feel free to keep using new tokens if that's the case 👍

Reply:
@codearos Could you please review if this change is alright? Thanks!